### ISSUE TITLE
package rebase: missing file paths-from-unix-to-windows

### ISSUE THREAD

**Issue by h-c-mueller** [Alexpux/MSYS2-packages#852]
File paths-from-unix-to-windows in package rebase used by autorebasebase1st.bat is missing, i.e. currently not effect of autorebasebase1st.bat.
It seems PKGBUILD should contain also
cp -f ${srcdir}/paths-from-unix-to-windows ${pkgdir}/usr/bin

**Comment by elieux** [comment]
Yeah, this script is experimental AFAIK. Should we remove it, @mingwandroid?

**Comment by h-c-mueller** [comment]
Just an additional short comment: with paths-from-unix-to-windows in place, both files base-dlls-unix.txt and base-dlls.txt are identical for me. So if paths-from-unix-to-windows is actually not required, the final rebaseall should use base-dlls-unix.txt.
